src/core_init_atmosphere/mpas_init_atm_static.F

```diff
@@ -282,13 +283,17 @@ subroutine init_atm_static(mesh, dims, configs)
 !
 ! Set land use and soil category parameters for water and ice
 !
+! TODO: Should this be moved to be above the interpolation of land use and soil at? Or
+! is it fine here?
  surface_input_select0: select case(trim(config_landuse_data))
     case('USGS')
+       ! Should we grab these values from the index file?
        isice_lu = 24
        iswater_lu = 16
        ismax_lu = 24
        write(mminlu,'(a)') 'USGS'
     case('MODIFIED_IGBP_MODIS_NOAH')
+       ! Should we grab these values from the index file?
        isice_lu = 15
        iswater_lu = 17
        ismax_lu = 20
```

Comment:
The index file for the MODIS dataset contains the above values (`isice`, `iswater` and `category_max`).